Test/Utils/ComputeNode/Configuration.ps1

```diff
@@ -32,21 +32,34 @@ function New-CNMPluginConfigFile {
 
     Write-Log 'Creating CNM plugin config file'
 
+    switch ($Testbed.WinVersion) {
+        'v2016' {
```

Comment:
Maybe you could change `$Testbed.WinVersion` to be `2016` or `2019` without `v` at the beginning and than use it directly without `switch`

Reply:
No, because it is enum, and cannot be just digits.